### DISCUSSION TITLE
Having trouble sending TCP packets to the external network in 'Shared Mode(NAT)'.

### DISCUSSION THREAD

**Discussion by SiwonLim** [utmapp/UTM#6125] [UTM Help (macOS)]
**Describe the issue**  
<img width="256" alt="issue-network-mode" src="https://github.com/utmapp/UTM/assets/51152816/bfdd095a-bc38-46db-8461-6839da9bab75">

When I executed the idle scanning command with Nmap, <b><I>'nmap -Pn -p 80 -sI [test IP in external network] [target IP in local network]'</b></I>, I didn't receive any response from the external network.
<img width="998" alt="303222450-e53478c4-814d-4c35-ba00-b7592a50342d" src="https://github.com/utmapp/UTM/assets/51152816/8aa2df59-a4a2-456c-87f2-3d84f74bd098">

According to this discussion page (https://github.com/utmapp/UTM/discussions/5067), the 'Shared Mode' in Network Mode is supposed to be the same as 'NAT'. But there seems to be an issue where some packets can't reach the external network during idle scanning using Nmap. Specifically, when sending [SYN, ACK] packets through the TCP protocol, the packets couldn't reach the external network. I'm curious about how 'Shared Mode' operates on UTM.

**Configuration**  
* UTM Version: 4.2.5
* macOS Version: 14.1.1
* Mac Chip (Intel, M1, ...): Apple M1 
* VM: Kali 2023.2a

**Comment by osy** [comment]
This is implemented using Apple's vmnet framework which is largely a black box. We don't really know how it works internally.
